test/src/e2e_vm_tests/test_programs/should_pass/return_in_strange_positions/test.toml

```diff
@@ -0,0 +1,72 @@
+category = "compile"
```

Comment:
To actually run the tests `category = "run"` should be used along with the checks. Here is an example how this can be done: _/sway/test/src/e2e_vm_tests/test_programs/should_pass/language/implicit_casting/test.toml_